src/Presenters/IncidentPresenter.php

```diff
@@ -73,6 +73,50 @@ public function created_at_iso()
     }
 
     /**
+     * Present diff for humans date time.
+     *
+     * @return string
+     */
+    public function scheduled_at_diff()
```

Comment:
It should be but @joecohens originally added them like this. I keep meaning to change them. I'll do it in another PR once merged.